### ISSUE TITLE
System default aliases (abuse, admin, postmaster)

### ISSUE THREAD

**Issue by gumida** [mail-in-a-box/mailinabox#1660]
Hello,
usually mailinabox is adding three aliases for every mail domain you are adding to the config:
admin@mydomain.com, abuse@mydomain.com and postmaster@mydomain.com.

I've noticed an issue (?) when working with wildcard configs:
When only "@mydomain.com" is configured for an domain, mailinabox is not adding these aliases and/or deleting existing ones. As soon es one additional alias is configured, mailinabox as adding these three defaults again. So, for example:

Config A:
> @mydomain.com -> UserMailbox

Config B:
> @mydomain.com -> UserMailbox
> dummy@mydomain.com -> UserMailbox
> abuse@mydomain.com -> AdminMailbox
> admin@mydomain.com -> AdminMailbox
> postmaster@mydomain.com -> AdminMailbox

I'm not sure about this, but shouldn't the admin entries always exist?

**Comment by alento-group** [comment]
I think that the logic here is as follows ... 

If mail for domain1.com is being forwarded to domain1.net, then there is no need for the 'default aliases' on domain1.net as they exist on domain1.com.  I would think that adding a user/alias triggers this because now there is an ACTUAL user on the domain rather than the domain solely being an alias for another domain.

**Comment by JoshData** [comment]
Yeah it sounds like a little bug, but it doesn't sound like the behavior is actually a problem, so it may not be worth changing.
